test/forms/WhyReturnSubmittedLateFormSpec.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import base.SpecBase
+import play.api.data.FormError
+
+class WhyReturnSubmittedLateFormSpec extends SpecBase {
+  val form = WhyReturnSubmittedLateForm.whyReturnSubmittedLateForm()
+
```

Comment:
can we add a test for successful binding?
```suggestion
  "WhyReturnSubmittedLateForm" should {
    "bind when a valid reason is given" in {
    ....
    }
```